acrobat/scripts/frictionless.js

```diff
@@ -12,16 +12,24 @@ export default function init(verb) {
 
   // Review Alloy
   if (reviewBlock) {
+    reviewAlloy();
     const reviewWait = setInterval(() => {
       const reviewForm = document.querySelectorAll('.hlx-Review');
       if (reviewForm.length > 0) {
         clearInterval(reviewWait);
         reviewForm[0].addEventListener('submit', (e) => {
           const data = Object.fromEntries(new FormData(e.target).entries());
           // verb, rating, comment
-          reviewAlloy();
           reviewFeedbackAlloy(verb, data.rating, data['rating-comments']);
         });
+        if (document.querySelectorAll('.tooltip').length > 0) {
+          document.querySelectorAll('.tooltip')[3].addEventListener('click', () => {
+            reviewFeedbackAlloy(verb, '4');
+          })
+          document.querySelectorAll('.tooltip')[4].addEventListener('click', () => {
+            reviewFeedbackAlloy(verb, '5');
+          })
+        }
```

Comment:
It would be better to call querySelectorAll() just one time under reviewBlock element that is predeclared to save cost and avoid getting a wrong element in case there are more elements with `tooltip` class.
```suggestion
        const reviewTooltip = reviewBlock.querySelectorAll('.tooltip');
        if (reviewTooltip.length > 0) {
          reviewTooltip[3].addEventListener('click', () => {
            reviewFeedbackAlloy(verb, '4');
          })
          reviewTooltip[4].addEventListener('click', () => {
            reviewFeedbackAlloy(verb, '5');
          })
        }
```